src/web/modules/transactions/transactions.http.ts

```diff
@@ -77,3 +79,44 @@ export async function show(req: Request, res: Response, next: NextFunction): Pro
     next(error)
   }
 }
+
+export async function statistics(req: Request, res: Response, next: NextFunction): Promise<void> {
+  try {
+    let account
+    const accountId = req.query.account
+
+    if (req.query.account) {
+      account = await AdminUsers.gatewayAccountServices(accountId)
+    }
+
+    const periodKeyMap: {[key:string]: string} = {
+      today: 'day',
+      week: 'week',
+      month: 'month'
+    }
+    const selectedPeriod: any = req.query.period || 'today'
```

Comment:
This should be moved to a method that can be unit tested, happy for that to happen in another PR!